### DISCUSSION TITLE
谁知道那个网盘,无会员,可以下载速度到10M/s以上

### DISCUSSION THREAD

**Discussion by shaoyang778899** [AlistGo/alist#5645] [Q&A]
如题,测试了好几个,速度都特别慢,哪位大佬知道可以告诉一下吗?主要用于下载文件.谢谢.

**Comment by fly-pigawa** [comment]
123盘可以，免费给2t不限速，但直链相关功能是收费点，也就是说挂alist费劲

**Comment by caoronglin** [comment]
天翼云盘完全可以

**Comment by ddns6** [comment]
天翼云盘、移动云盘、123盘 此三款都可以。
另外，我的阿里挂载也行。
